rm/rm-policy/rm-policy-scheduler/src/main/java/org/ow2/proactive/scheduler/resourcemanager/nodesource/policy/EC2Policy.java

```diff
@@ -1,174 +0,0 @@
-/*
```

Comment:
You have deleted the CronSlotLoadBasedPolicy and the EC2Policy, so the SchedulerLoadingPolicy replaces them, correct?